client/src/utils/Cranefs.ts

```diff
@@ -101,21 +117,47 @@ export class Cranefs {
         });
     }
 
-    public processProject() {
+    public processProject(): void {
         Debug.info('Building project from cache file: ' + this.getTreePath());
         Crane.langClient.sendRequest({ method: "buildFromProject" }, {
             treePath: this.getTreePath(),
             saveCache: this.isCacheable()
         });
     }
 
-    public rebuildProject() {
+    public rebuildProject(): void {
         Debug.info('Rebuilding the project files');
         fs.unlink(this.getTreePath(), (err) => {
             this.processWorkspaceFiles(true);
         });
     }
 
+    public downloadPHPLibraries(): void {
+        var zip = Config.phpstubsZipFile;
+        var tmp = this.getCraneDir() + '/phpstubs.tmp.zip';
+        Debug.info(`Downloading ${zip} to ${tmp}`);
+        this.createCraneFolder().then((created) => {
+            fs.unlink(this.getStubsDir(), (err) => {
+                this.createPhpStubsFolder().then((stubsCreated) => {
+                    if (stubsCreated) {
+                        var file = fs.createWriteStream(tmp);
+                        http.get(zip, (response) => {
+                            response.pipe(file);
+                            response.on('end', () => {
+                                Debug.info('PHPStubs Download Complete');
+                                Debug.info(`Unzipping to ${this.getStubsDir()}`);
+                                fs.createReadStream(tmp)
+                                    .pipe(unzip.Parse())
+                                    .pipe(fstream.Writer(this.getStubsDir()));
+                                window.showInformationMessage('Libraries dowloaded and unziped. You might need to rebuild php sources.');
```

Comment:
Can this message be changed to `PHP Library Stubs downloaded and installed. You might need to rebuild the PHP sources for them to work correctly.` ?